<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceInfoWidget</class>
 <widget class="QWidget" name="DeviceInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>71</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>71</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="deviceRadioBtn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deviceNameLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>设备IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deviceIPLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>加卸载进度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="operationProgressLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
